import/trello/importTrello.ts

```diff
@@ -135,6 +136,29 @@ function convert(input: Trello): Block[] {
 
             outCard.fields.contentOrder = [text.id]
         }
+
+        // Add Checklists
+        if (card.idChecklists && card.idChecklists.length) {
+            card.idChecklists.forEach(checklistID => {
+                const lookup = input.checklists.find(e => e.id === checklistID)
+                if (lookup !== undefined) {
+                    lookup.checkItems.forEach(trelloCheckBox=> {
+                        const checkBlock = createCheckboxBlock()
+                        checkBlock.title = trelloCheckBox.name
+                        if (trelloCheckBox.state == 'complete') {
```

Comment:
The eslint setup for the import scripts may be out of date from the webapp, but in general we should use ===, i.e. `if (trelloCheckBox.state === 'complete') {`.